content/how-to/add-code-servers/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Adding code servers"
-description: "How to add code servers to your tenant"
-lead: "How to add code servers to your tenant"
```

Comment:
Actually, this was a good lead and description. They're different from the title and they accurately describe the page. I wouldn't remove these.